v3/src/components/data-display/hooks/use-data-tips.ts

```diff
@@ -73,11 +74,12 @@ export const useDataTips = ({dotsRef, dataset, displayModel}: IUseDataTips) => {
 
     // support disabling data tips via url parameter for jest tests
     if (urlParams.noDataTips === undefined) {
-      dotsRef.current && select(dotsRef.current)
-        .on('mouseover', showDataTip)
-        .on('mouseout', hideDataTip)
-        .call(dataTip)
+      // TODO PIXI: reimplement data tips
+      // dotsRef.current && select(dotsRef.current)
+      //   .on('mouseover', showDataTip)
+      //   .on('mouseout', hideDataTip)
+      //   .call(dataTip)
```

Comment:
Is there a PT story for this so we don't lose track of it?

Reply:
Yes: https://www.pivotaltracker.com/story/show/186725308